client/app/pages/dashboards/DashboardPage.jsx

```diff
@@ -19,6 +19,7 @@ import url from "@/services/url";
 import useImmutableCallback from "@/lib/hooks/useImmutableCallback";
 
 import useDashboard from "./hooks/useDashboard";
+import addToRecentsAtLocalStorage from "../../lib/addToRecentsAtLocalStorage";
```

Comment:
```suggestion
import addToRecentsAtLocalStorage from "@/lib/addToRecentsAtLocalStorage";
```

Also, please run Prettier over changed files

Reply:
Ok.